redis_bitset.go

```diff
@@ -1,28 +1,33 @@
 package bloom
 
 import (
+	"fmt"
 	"github.com/garyburd/redigo/redis"
+	"strings"
 )
 
+const REDIS_MAX_LEN = 8 * 512 * 1024 * 1024
```

Comment:
consts in Go still follow the camel-case rule -- all the stdlib counter-examples have been publicly regret'd by the designers and the core team. At any rate, there's no reason for this to be exported that I can see -- it's an internal implementation detail.